typescript/widgets/src/chains/ChainAddMenu.tsx

```diff
@@ -155,14 +155,7 @@ function tryParseMetadataInput(
     return failure('name is already in use by another chain');
   }
 
-  if (multiProvider.tryGetChainMetadata(newMetadata.chainId)) {
-    return failure('chainId is already in use by another chain');
-  }
```

Comment:
honestly im not sure we should remove this constraint from the multiprovider as a signer can think they are signing for chain X but their transaction can be replayed on chain Y

Reply:
I think this is probably fine for us because my understanding is with our model we want to make sure that if we have a chain X and chain Y with the same chain ID, they are both actually chain X but just with different domain X and domain Y